src/dos/program_mount_common.h

```diff
@@ -31,6 +31,8 @@
 
 extern Bitu ZDRIVE_NUM;
 
+constexpr int minimum_length = 11;
```

Comment:
With this floating on its own in the header, it feels a bit disconnected ("length" in this case might mean something about the drives being mounted, as opposed to column widths).  

Suggest adding a comment or naming it more specifically to explain/describe that this is a column width minimum length? (or maybe it's not a column width.. I don't actually know.. but mentioning it as an example).

Reply:
The header applies to both IMGMOUNT and MOUNT commands, which was why I moved it here. Now I have changed the name to be more descriptive and also added a comment.